docs/application/toc_all.md

```diff
@@ -89,6 +89,7 @@
 ##### [TableView](/application/dotnet/guides/nui/tableview.md)
 ##### [Text](/application/dotnet/guides/nui/text.md)
 ##### [VideoView](/application/dotnet/guides/nui/videoview.md)
+##### [Window](/application/dotnet/guides/nui/window.md)
```

Comment:
Question to the Window.

In this tree, as you can see other items are under Tizen.NUI.Components, according to the API page.

However, it seems that the Window is not under Tizen.NUI.Components, but in other category as Tizen.NUI.Window. 
(I'm not sure, as I don't get the exact API page for the Window.)

Would you please check once again if this location is proper for the Window?

Reply:
i agree. so i will change toc_all.md